src/Sylius/Behat/Context/Setup/CartContext.php

```diff
@@ -188,4 +194,9 @@ private function addProductToCart(ProductInterface $product, ?string $tokenValue
 
         $this->sharedStorage->set('product', $product);
     }
+
+    private function doesCartWithTokenExist(string $tokenValue): bool
```

Comment:
```suggestion
    private function cartWithTokenExists(string $tokenValue): bool
```
But if you leave it as it is, it will also be fine